src/components/claim/views/success.view.tsx

```diff
@@ -35,25 +35,12 @@ export function ClaimSuccessView({ txHash, claimDetails, senderAddress }: _const
     useEffect(() => {
         router.prefetch('/send')
         gaEventTracker('peanut-claimed', 'success')
-        sendNotification(claimDetails[0])
+        sendNotification()
     }, [])
 
-    const sendNotification = async (linkDetails: interfaces.ILinkDetails) => {
-        console.log('sendNotification', senderAddress)
-        const accounts = [`eip155:1:${senderAddress}` ?? '']
-        const chainName = chainDetails.find(
-            (detail) => detail.chainId.toString() === linkDetails.chainId.toString()
-        )?.name
-        const notification = {
-            title: 'Peanut Protocol',
-            body: `Your link has been claimed on ${chainName} by ${address ?? ''}`,
-            url: undefined,
-            type: '2aee6e5f-091d-444e-96cd-868ba2ddd0e7',
-        }
-        utils.sendNotification({
-            notification,
-            accounts,
-        })
+    const sendNotification = async () => {
+        const chainName = chainDetails.find((detail) => detail.chainId === claimDetails[0].chainId)?.name
```

Comment:
Given that chainDetails doesn't update on success view, and this line is repeated: `chainDetails.find((detail) => detail.chainId === claimDetails[0].chainId)?.name`

Why not make it a const?

Reply:
Honestly, dunno. From what I've found, chainDetails in UI is always used this way: `const [chainDetails] = useAtom(store.defaultChainDetailsAtom)`, and I thought it's better to not make any changes to the approach without Bo.